### PR TITLE
[Lang] Fix gui docstring

### PR DESCRIPTION
**Pull request by neozhaoliang**
Close: #6999

### CODE REVIEW

File: python/taichi/ui/canvas.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def circles(self,
         Args:
             centers: a taichi 2D Vector field, where each element indicate the \
                 3D location of a vertex.
-            radius (float): radius of the circles, relative to the height of the screen.
+            radius (Union[float, numpy.array]): radius of the circles in pixel.
```
Comment (ailzhang) [inline python/taichi/ui/canvas.py:114]: nit: do we limit the dimension of the numpy array?

File: python/taichi/ui/canvas.py [hunk 2]
```diff
@@ -111,7 +111,9 @@ def circles(self,
         Args:
             centers: a taichi 2D Vector field, where each element indicate the \
                 3D location of a vertex.
-            radius (float): radius of the circles, relative to the height of the screen.
+            radius (Union[float, numpy.array]): radius of the circles in pixels. \
+                Can be either a single number, which will be applied to all circles, or a 1D NumPy array of the same length as `centers`.\
+                The `i-th` item in the array will
```
Comment (ailzhang) [inline python/taichi/ui/canvas.py:116]: nit: unfinished sentence?

Comment (neozhaoliang) [reply python/taichi/ui/canvas.py]: Sorry, that's so awkward....

**Review by ailzhang — APPROVED** [review]
LGTM!

**Comment by YouJiacheng** [comment]
According to the shader
https://github.com/taichi-dev/taichi/blob/3305acc76dade9844a9617648d9931940902d1c9/python/taichi/shaders/Circles_vk.vert

I don't think that GGUI support array/field `radius`.

**Comment by neozhaoliang** [comment]
@YouJiacheng Thanks for pointing out this. The `GUI.circles` does support passing a numpy array for the radius, but `GGUI.circles` doesn't. This is will be fixed in another PR #7065

**Pull request merged**